app/api/schema/__init__.py

```diff
@@ -0,0 +1,11 @@
+# Monkey Patch Marshmallow JSONAPI
+from marshmallow_jsonapi.flask import Relationship
+
+
+def serialize(self, attr, obj, accessor=None):
```

Comment:
@iamareebjamal @prateekj117 This change has broke the user endpoint

Reply:
This didn't